frontend/src/components/campoComBuscaRemota/CampoComBuscaRemota.vue

```diff
@@ -94,19 +97,24 @@
             :key="item.id"
           >
             <slot
-              name="tableData"
+              name="table-data"
               v-bind="{ item }"
             >
-              {{ item[chaveDeExibição] }}
+              <td>
+                {{ item[chaveDeExibicao] }}
+              </td>
             </slot>
+
             <td>
-              <button
-                type="button"
-                class="btn bgnone outline tcprimary"
-                @click="selecionarItem(item)"
-              >
-                Selecionar
-              </button>
+              <div class="flex justifyright">
```

Comment:
Para que esse `div`, @GustavoFSoares ?! Se for só para alinhar, dá para fazer com margem. Sem markup extra.

Reply:
@Eduruiz

Reply:
@robsonsobral e so pra manter o botão sempre a ao final mesmo, acho que pra alinhar componentes sempre e melhor controlar num envelope mesmo que gere um elemento de markup a mais.